propelauth_py/api/user.py

```diff
@@ -14,36 +14,38 @@
     RateLimitedException,
 )
 
-ENDPOINT_PATH = "/api/backend/v1/user"
+ENDPOINT_URL = f"{BACKEND_API_BASE_URL}/api/backend/v1/user"
```

Comment:
All instances of `ENDPOINT_PATH` now become `ENDPOINT_URL` and don't have to concatenate w/ the auth_url anymore